docs/components/remote-view.md

```diff
@@ -0,0 +1,296 @@
+
+```jsx
+// Use an iframe fallback for one particular view
+function determineFallbackCases(manifest: MicrofrontendManifest) {
+  if (manifest.name === '@myscope/example-module') {
+    return true;
+  }
+
+  return false;
+}
+
+return (
+  <RemoteViewProvider>
+    {remoteViews.map((url, key) => (
+      <section key={key}>
+        <RemoteViewErrorBoundary>
+          <RemoteView
+            url={url}
+            loadWithIframeFallback={determineFallbackCases}
+          />
+        </RemoteViewErrorBoundary>
+      </section>
+    ))}
+  </RemoteViewProvider>
+);
+```
+
+## Custom loader
+
+You can customize the loading state of each `<RemoteView />` by providing a
+`loading` prop:
+
+```jsx
+return (
+  <RemoteViewProvider>
+    <RemoteViewErrorBoundary>
+      <RemoteView
+        url={myUrl}
+        loadWithIframeFallback={determineFallbackCases}
+        loading={<div>My custom loading component</div>}
+      />
+    </RemoteViewErrorBoundary>
+  </RemoteViewProvider>
+);
+```
+
+By default, `<RemoteView />` will output a very simple loading message:
+`<div>Loading</div>`.
+
+Alternatively, you bring your own error boundary.
+
+## Security
+
+Loading dynamic (especially remote, over http(s)) ES modules at runtime carries
+certain risks and must be used carefully. Content loaded dynamically should only
+ever be done from a trusted source.
+
+End user input should **not** be used to determine content URLs (etc) and any
+application of consuming modules at runtime should be centrally controled in a
+secure environment. In other words, user input should not be allowed to modify
+URLs that `<RemoteView />` will attempt to render.
+
+There is also no guarantee of safety, which is why it is very strongly
+recommended to use error boundaries. `<RemoteView />` attempts to render
+whatever module it finds into your React tree, and this depends on runtime
+imports actually returning valid React modules.
+
+## CSS
+
+`<RemoteView />` automatically imports local and global CSS from ESM Views that
+have been specified according to the
+[Modular docs](https://modular.js.org/esm-views/external-css-imports/).
+
+When an iframe fallback is used, this does not apply.
+
+## Error handling
+
+It is strongly recommended to wrap any usage of `<RemoteView />` with error
+boundaries. There are two approaches you can take:
+
+- Use the provided `<RemoteViewErrorBoundary />`, optionally customizing the
+  content that is displayed
+- Use your own error boundary
+
+### Using the default error boundary
+
+Outputs a default, simple error component displaying the error details
+
+```jsx
+return (
+  <RemoteViewProvider>
+    <RemoteViewErrorBoundary>
+      <RemoteView url={url} />
+    </RemoteViewErrorBoundary>
+  </RemoteViewProvider>
+);
+```
+
+### Customizing the content of `<RemoteViewErrorBoundary />`
+
+Supply the `content` prop. If the `error` prop is an instance of
+`RemoteViewError`, will contain RemoteView-specific error details.
```

Comment:
Which are these details? We should mention `remoteViewUrl` here and in the code example below

Reply:
An updated example that demonstrates this is on the way